README.md

```diff
@@ -91,13 +184,108 @@ You then need to do a second call to the `signIn` mutation providing only the `o
 }
 ```
 
-## OAuth
+## OAuth / Login with Sorare
+
+If you want to make requests on behalf of other users you can rely on the [OAuth](https://oauth.net/2/) protocol.
+
+Please reach out to us first through our [Help Center](https://help.sorare.com/hc/en-us/requests/new) and ask us to generate you some OAuth credentials.
+
+We need:
+
+- the name of your App
+- the callback URIs of your OAuth applications
+  - ex: `http://localhost:3000/auth/sorare/callback` (for dev) & `https://myapp.com/auth/sorare/callback` (for prod)
+- the PNG version of your App logo
+
+### OAuth Credentials
+
+Once manually validated, we'll provide you with:
+
+- OAuth UID (or Client ID)
+- OAuth Secret (keep this secret!)
+
+### OAuth Scopes
+
+No specific scopes are available at the moment. All OAuth apps will be have the same privileges.
```

Comment:
All OAuth applications are provided with one scope which allows access to the following:
- Basic user information, including their nickname, avatar, and wallet address
- User's cards, achievements and favorites
- User's auctions, offers and notifications